docs/examples/tutorial/cython_tutorial/primes.py

```diff
@@ -0,0 +1,27 @@
+def primes(nb_primes: cython.int):
+    i: cython.int
+    p: cython.int[1000]
+
+    if nb_primes > 1000:
+        nb_primes = 1000
+
+    if not cython.compiled:
+        p = [0] * 1000
+
+    len_p: cython.int = 0  # The current number of elements in p.
+    n: cython.int = 2
+    while len_p < nb_primes:
+        # Is n prime?
```

Comment:
This is probably a good time to fix this comment (do the same to `primex.pyx`)
```suggestion
        # Is n a prime?
```

Reply:
Well, ok. We also could add "number", idk is it worth is.
"Is n a prime number?"
@scoder what do you think?

Reply:
I think it's correct in mathematics to use "prime" as an adjective. It's either "n is a prime number" or "n is prime".

Reply:
In general, I think the most important thing is for the reader to understand what is at stake.